examples/kustomization/sts-example/sample-data/mc-job-setup-bucket.yaml

```diff
@@ -52,7 +52,7 @@ spec:
           env:
             - name: MC_HOST_local
```

Comment:
just a thought, perhaps we should use the alias `myminio` to not confuse people as what is local here

Reply:
sure!